heft-plugins/heft-sass-plugin/src/SassTypingsGenerator.ts

```diff
@@ -1,12 +1,12 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
-import * as path from 'path';
+import { LegacyAdapters } from '@rushstack/node-core-library';
```

Comment:
We tend to order using the following priority (alphabetically within each):
- system dependencies
- external dependencies (declared in `package.json`, obtained from NPM)
- monorepo dependencies (declared in `package.json`, obtained from monorepo workspace dependency)
- internal module dependencies (included in package)

Though I don't think that this is documented anywhere (or even guidance.... @octogonz ?)

Reply:
Ok, I've reordered the imports now.

Reply:
Doesn't seem like they were re-ordered? Should look largely like the original, with the `import * as path from 'path'` at the top and the `@rushstack` imports at the bottom. If this is happening because of the prettier-on-commit functionality of the repo then you can ignore this. But it would be odd if so, since we use this format pretty widely across the repo.

Reply:
Sorry, only fixed `SassTypingsPlugin.ts` earlier. This file should be fixed now as well.